src/components/Explore/utils.ts

```diff
@@ -430,6 +435,21 @@ export const exploreMetricData: {
       },
     ],
   },
+  [ExploreMetric.WEEKLY_DEATHS_PER_100K]: {
+    title: 'Weekly reported deaths per 100k',
+    name: 'Weekly reported deaths per 100k',
+    chartId: 'weekly_reported_deaths_per_100k',
```

Comment:
I don't see "reported" in the metric name in the [copy doc](https://docs.google.com/document/d/1pITjM1gctulpwyqez5SFrQLFIYO4vndJArDjFPnVMG4/edit#). I think there is perhaps an ongoing discussion about whether we want "weekly cases" or "weekly reported cases" since many cases won't be reported.  But I don't think that's a significant concern for deaths.  Did you see something that said this should have "reported" in it?

Reply:
Nope! This can be removed.